CHANGELOG.md

```diff
@@ -1,26 +1,16 @@
 # Changelog
 
-## v1.1.15 - UNRELEASED
+## v1.1.13 - 2025-02-26
 
 ### Added
 
 - **aiken-lsp**: an additional code action to use constructors or identifiers from qualified imports is now offered on missing constructor or identifier. @KtorZ
 
 ### Changed
 
-- **aiken-lang**: fixed `UnknownTypeConstructor` wrongly reported as `UnknownVariable` (then messing up with LSP quickfix suggestions). @KtorZ
```

Comment:
This didn't happe in 1.13  though, nor the above. They haven't been released yet. 

I don't see what's wrong with the current CHANGELOG actually...